data/templates/photon-os-ks

```diff
@@ -0,0 +1,35 @@
+{
+    "hostname": "test_hostname",
+    "password":
+        {
+            "crypted": false,
```

Comment:
when set crypted to ture, the following password text will be crypted, which make it hard for user to set a password.